addons/website_slides/static/src/scss/slides_slide_fullscreen.scss

```diff
@@ -115,6 +115,16 @@
             pointer-events: none;
         }
     }
+
+    .o_wslides_button_complete {
+        height: 25px;
+    }
```

Comment:
Could perhaps be replace with bootstrap class like h-50 or h-25 ? Tell me what you think ?